pkg/processor/document/tika_client.go

```diff
@@ -5,16 +5,25 @@ import (
 )
 
 type TikaRoundTripper struct {
-	r http.RoundTripper
+	r                   http.RoundTripper
+	ocrLanguageOverride string
 }
 
 // https://cwiki.apache.org/confluence/display/tika/TikaJAXRS#TikaJAXRS-MultipartSupport TIKA must have an Accept header to return JSON responses.
 func (mrt TikaRoundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
 	if r.URL.Path == "/meta" {
 		r.Header.Add("Accept", "application/json")
+		r.Header.Add("X-Tika-OCRLanguage", "ces")
```

Comment:
Shouldn't the hard-coded `ces` here be a reference to the `ocrLanguageOverride` parameter?

Reply:
🤦 You are absolutely right. This is what rushing PR late at night gets you.

It is a remain of some testing attempt I did. AFAIK it should be enough to pass the OCR language to `/tika`. Metadata retrieval seems unaffected by it.

I have removed that line.

Reply:
Yeah, totally understand the late night thing. No worries, that's what the PR review process is for. I'm happy I was able to understand what the code was doing enough to catch it even though I've never written a line of Go code in my life :)